components/lib/autocomplete/AutoCompletePanel.js

```diff
@@ -12,6 +12,16 @@ export const AutoCompletePanel = React.memo(
             return ObjectUtils.resolveFieldData(optionGroup, props.optionGroupLabel);
         };
 
+        const createFooter = () => {
+            if (props.panelFooterTemplate) {
+                const content = ObjectUtils.getJSXElement(props.panelFooterTemplate, props, props.onOverlayHide);
+
+                return <div className="p-multiselect-footer">{content}</div>;
```

Comment:
This should be `p-autocomplete-footer` class name

Reply:
Fixed it.